binding.gyp

```diff
@@ -76,6 +76,7 @@
             ['OS != "win"', {
               'libraries': [
                 '<(module_root_dir)/build/libzmq/lib/libzmq.a',
+                "<!@(pkg-config libsodium --libs)",
```

Comment:
This should not be needed here since it is handled inside CMake.
Do you get an error without this?

Reply:
I know. But somehow it is needed for node-gyp in order to resolve symbols properly!
(In older build systems)
I verified this on the setup described above.

Reply:
Hi, I can confirm that this solved my issue with compiling native library for usage with electron (requires build against specific node version) on Linux (up-to-date Manjaro/Arch). Is there any chance to have this fix in official package? Cheers, Martin